docs/source/concept_guides/quantization.mdx

```diff
@@ -1,5 +1,256 @@
+
+#### Calibration
+
+The section above described how quantization from `float32` to `int8` works, but one question
+remains: how is the `[a, b]` range of `float32` values determined? That is where calibration comes in to play.
+
+Calibration is the step during quantization where the `float32` ranges are computed. For weights it is quite easy since
+the actual range is know at *quantization-time*. But it is less clear for activations, and different approaches exist:
+
+1. Post training dynamic quantization: the range for each activation is computed at *runtime*. While this gives great
+results without too much work, it can be a bit slower than static quantization because the range is computed each time.
+It is also not an option on certain hardware.
+2. Post training static quantization: the range for each activation is computed at *quantization-time*.
+  - Observers are put on activations to record their values.
+  - A certain number of forward passes on a calibration dataset is done (around `200` examples is enough).
+  - The ranges for each computation are computed according to some *calibration technique*.
+3. Quantization aware training: the range for each activation is computed at *training-time*, following the same idea
+than post training static quantization. But "fake quantize" operators are used instead of observers: they record
+values just as observers do, but they also simulate the error induced by quantization to let the model adapt to it.
+
+
+For both post training static quantization and quantization aware training, it is necessary to define calibration
+techniques, the most common are:
+
+- Min-max: the computed range is `[min observed value, max observed value]`, this works well with weights.
+- Moving average min-max: the computed range is `[moving average min observed value, moving average max observed value]`,
+thiw works well with activations.
+- Histogram: records a histogram of values along with min and max values, then chooses according to some criterion.
+  - Entropy: the range is computed as the one minimizing the error between the full-precision and the quantized data.
+  - Mean Square Error: the range is computed as the one minimizing the mean square error between the full-precision and
+  the quantized data.
+  - Percentile: the range is computed using a given percentile value `p` on the observed values. The idea is to try to have
+  `p%` of the observed values in the computed range. While this is possible when doing affine quantization, it is not always
+  possible to exactly match that when doing symmetric quantization. You can check [how it is done in ONNX
+  Runtime](https://github.com/microsoft/onnxruntime/blob/2cb12caf9317f1ded37f6db125cb03ba99320c40/onnxruntime/python/tools/quantization/calibrate.py#L698)
+  for more details.
+
+
+#### Pratical steps to follow to quantize a model to `int8`
+
+To effectively quantize a model to `int8`, the steps to follow are:
```

Comment:
probably the tradeoff speed / prediction quality should be highlighted here.

![image](https://user-images.githubusercontent.com/9808326/207072097-27e7d71b-5a92-4df4-abb8-e75df9131e99.png)